book.toml

```diff
@@ -6,7 +6,7 @@ src = "src"
 title = "Breez SDK"
 
 [output.html]
-additional-js = ["tabs.js"]
```

Comment:
It's better to put the new highlight.js in a directory called "theme" and add theme="theme"because right now two files name highlight.js are loaded

Reply:
dart is supported now by 680f9b7b569923b20798002b86e1483ff1bbb13c